### DISCUSSION TITLE
Quick question about 5V in/out, USB and SSR relays

### DISCUSSION THREAD

**Discussion by HalbesHaehnchen** [phil-barrett/grblHAL-teensy-4.x#90] [Q&A] [answered]
Soon (I hope) I assembled my machine, and I am already using my "waiting for the post man" time to study the board I bought.
Plan is, to use 3x  SSR relays to activated exhaust fan, air compressor and a tiny 24v DC vacuum. 
Now the board is powered by the USB connection to my PC. 

The big relay terminal array is for powering the relays right? So I set a jumper to 5v.  Can the board handle 3 5v SSRs ? Or do I have to cut those 5v USB Pads on the downside and connect an external 5v power supply to the 5v Terminal?

Thanks in advance

**Comment by phil-barrett** [comment, marked as answer]
You don't need to use the jumpers if you are directly using the TTL headers. They are always 5V. The board will handle them fine from USB power. 

If you want to use the screw terminals, look here. https://www.grbl.org/single-post/ttl-ssr-on-the-screw-terminals

**Comment by HalbesHaehnchen** [comment]
How about the aux 0 relays connector? I found out I can activate it with "m64 p0", but how to deactivate it? It stays on even after reset. Usually I wouldnt need the extra relay outputs but flood and mist both gets deactivated by m9 command, that gives me some problems.

**Reply by karoria** [reply]
M65 P0 switches off that output immediately.

**Reply by HalbesHaehnchen** [reply]
thanks that worked.

**Reply by phil-barrett** [reply]
The LinuxCNC documentation describes the [M62-M65](https://linuxcnc.org/docs/html/gcode/m-code.html#mcode:m62-m65), related to Aux0-Aux2.
